.github/workflows/browserstack.yml

```diff
@@ -0,0 +1,35 @@
+name: BrowserStack
+on: [push]
+
+jobs:
+  browserstack:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 3
+
+      - name: Set Node.js version
+        uses: actions/setup-node@v1
+        with:
+          node-version: "10"
+
+      - run: node --version
+      - run: npm --version
+
+      - name: Install npm dependencies
+        run: npm ci
+        env:
+          CI: true
+
+      - name: Run dist
+        run: npm run dist
+
+      - name: Run BrowserStack tests
+        run: npm run js-test-cloud
+        if: github.repository == 'twbs/bootstrap'
```

Comment:
Not sure if this check makes sense anymore?

Reply:
Why was this added in the first place?